draft-quic-atsss-reqs.mkd

```diff
@@ -318,7 +318,7 @@ To avoid such issue, the ATSSS UPF may be configured with a pool of IP addresses
 - Access Traffic Switching: migration of all packets of an ongoing data flow from one access network to another access network while ensuring session continuity. 
 - Access Traffic Splitting: split of the packets of a data flow across multiple access networks.
 
-Techniques to provide ATSSS are classified by the 3GPP into two flavors: (1) high-layer techniques which operate above the IP layer (e.g., MPTCP), and (2) low-layer techniques which operate below the IP layer. Only high-layer steering techniques are discussed in this document.
+Techniques to provide ATSSS are classified by the 3GPP into two flavors: (1) high-layer techniques which operate above the IP layer (e.g., MPTCP), and (2) low-layer techniques which operate below the IP layer. 
 
```

Comment:
This is still confusing because in lower layer QUIC approach you have two IP layers...

Reply:
I would just remove this whole sentence here.

Reply:
No sure about this. We are echoing 23501.

"
-	High-layer steering functionalities, which operate above the IP layer:
-	In this release of the specification, only one high-layer steering functionality is specified, which applies the MPTCP protocol [81], and is called "MPTCP functionality" (see clause 5.32.6.2.1). This steering functionality can be applied to steer, switch and split the TCP traffic of applications allowed to use MPTCP. The MPTCP functionality in the UE may communicate with an associated MPTCP Proxy functionality in the UPF, by using the MPTCP protocol over the 3GPP and/or the non-3GPP user plane.
-	Low-layer steering functionalities, which operate below the IP layer:
-	One type of low-layer steering functionality defined in the present document is called "ATSSS Low-Layer functionality", or ATSSS-LL functionality (see clause 5.32.6.3.1). This steering functionality can be applied to steer, switch and split all types of traffic, including TCP traffic, UDP traffic, Ethernet traffic, etc. ATSSS-LL functionality is mandatory for MA PDU Session of type Ethernet. In the network, there shall be in the data path of the MA PDU session one UPF supporting ATSSS-LL.
NOTE:	Filters used in ATSSS rules related with a MA PDU Session of type Ethernet can refer to IP level parameters such as IP addresses and TCP/UDP ports.

"

Reply:
Yes, it not wrong to say this, it just confusing because there are two IP layers in case of QUIC based proposal. To avoid this confusing I would rather avoid to talk about this at all in the IETF context because it's actually not important for the content of this document.

Reply:
If we remove that part, I'm afraid that the reader won't understand why are we talking about the following in our gaps section:

"ATSSS Phase 2 will need transport services for **Ethernet frames** and IP packets"

Let's leave the text and revise in the next iteration if we have comments. 

Thank you.